cmd/batchReportGenerator/main.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+//go:generate sh -c "mkdir -p ../../gen/reportsStorage/ && swagger generate client -A reportsStorage -t ../../gen/reportsStorage/ -f ../../docs/api/reportsStorage.yml --principal string"
+
+import (
+	"bytes"
+	"context"
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/go-openapi/runtime"
+	runtimeClient "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	"github.com/iryonetwork/wwm/storage/keyvalue"
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/push"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/reportsStorage/client"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/reportsStorage/client (cannot find package "github.com/iryonetwork/wwm/gen/reportsStorage/client" in any of:
	/tmp/tmp.jNUNwZeI6Z/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/reportsStorage/client (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/reportsStorage/client (from $GOROOT)
	/tmp/tmp.jNUNwZeI6Z/src/github.com/iryonetwork/wwm/gen/reportsStorage/client (from $GOPATH))